Output directory adapter lookup

singhsarab opened the pull request:
Changes for picking up adapters from the output directory in case testadapterpath is not given.

singhsarab committed:
Changes for picking up adapters from the output directory in case testadapterpath is not given.

singhsarab committed:
Merge branch 'master' into ProjectOutputDir

codito commented on src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs:
```diff
@@ -39,10 +41,12 @@ public class DefaultTestHostManager : ITestRuntimeProvider
 
         private const string DefaultTestHostUri = "HostProvider://DefaultTestHost";
         private const string DefaultTestHostFriendlyName = "DefaultTestHost";
+        private const string TestAdapterRegexPattern = @"TestAdapter.dll";
```
We moved away from regex to simple string match IIRC

codito commented on src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs:
```diff
@@ -176,6 +183,11 @@ public TestHostConnectionInfo GetTestHostConnectionInfo()
         /// <inheritdoc/>
         public IEnumerable<string> GetTestPlatformExtensions(IEnumerable<string> sources, IEnumerable<string> extensions)
         {
+            if (sources != null && sources.Any() && this.projectOutputExtensionsRequired)
```
Does this work for not isolation scenario?

singhsarab replied:
Yes.

singhsarab committed:
Moving the RunSettingsProviderExtensions to common.
Also moving the updation of the runsettings to include extensions folder to ProxyDataCollectorManager

singhsarab committed:
Merge branch 'ProjectOutputDir' of https://github.com/singhsarab/vstest into ProjectOutputDir

singhsarab committed:
Fixing the build break due to merge.

singhsarab committed:
Fixed the issue with innerXml not getting updated in case of dotnet core.

singhsarab committed:
Merge branch 'master' into ProjectOutputDir

singhsarab committed:
Small fix in the name as per comment.

singhsarab committed:
Merge branch 'ProjectOutputDir' of https://github.com/singhsarab/vstest into ProjectOutputDir

smadala commented on test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs:
```diff
@@ -30,6 +30,19 @@ public void RunMultipleTestAssemblies(string runnerFramework, string targetFrame
+            AcceptanceTestBase.SetTestEnvironment(this.testEnvironment, runnerFramework, targetFramework, targetRuntime);
+
+            var assemblyPaths =
+                this.BuildMultipleAssemblyPath("SimpleTestProject.dll", "SimpleTestProject2.dll").Trim('\"');
```
Pass xunit dll as second dll.

smadala replied:
Remove /testAdapterPath:"$testAdapterPath" in test.ps1.

smadala commented:
Fixes: https://github.com/Microsoft/vstest/issues/251

singhsarab committed:
Merge remote-tracking branch 'upstream/master' into ProjectOutputDir

singhsarab committed:
Modifying the test as per the suggestion.
